go-controller/pkg/node/gateway_iptables.go

```diff
@@ -43,8 +43,14 @@ type iptRule struct {
 
 func addIptRules(rules []iptRule) error {
 	var addErrors error
+	var protocolStr string
 	for _, r := range rules {
-		klog.V(5).Infof("Adding rule in table: %s, chain: %s with args: \"%s\" for protocol: %v ", r.table, r.chain, strings.Join(r.args, " "), r.protocol)
+		if r.protocol == iptables.ProtocolIPv4 {
+			protocolStr = "ipv4"
+		} else {
+			protocolStr = "ipv6"
+		}
+		klog.V(5).Infof("Adding rule in table: %s, chain: %s with args: \"%s\" for protocol: %s ", r.table, r.chain, strings.Join(r.args, " "), protocolStr)
```

Comment:
why do we need this new string for logging errors? how does `r.protocol` look like?

Reply:
0 or 1. So the log message returned:
`Adding rule in table: %s, chain: %s with args: \"%s\" for protocol: 0`
or 
`Adding rule in table: %s, chain: %s with args: \"%s\" for protocol: 1`